config/policy.go

```diff
@@ -679,12 +699,14 @@ func (p *Policy) GetPassIdentityHeaders(options *Options) bool {
 }
 
 type routeID struct {
-	From     string
-	To       []string
-	Prefix   string
-	Path     string
-	Regex    string
-	Redirect *PolicyRedirect
+	From                 string
+	To                   []string
+	Prefix               string
+	Path                 string
+	Regex                string
+	Redirect             *PolicyRedirect
+	DirectResponseStatus int
+	DirectResponseBody   string
```

Comment:
Any reason not to use a single `*DirectResponse` field here, like the `*PolicyRedirect` line above?